vertigo-bot-docker/docker-compose.yml.sample

```
@@ -54,6 +54,10 @@ services:
       - INFLUXDB_URL=http://influxdb:8086
       - ANALYTICA_DBNAME=chatbot
       - DEPLOY_PATH=designer
+    volumes:
+      - type: bind
+        source: ./keycloak.json
+        target: /opt/keycloak/keycloak.json

```

peut être simplifié en
```suggestion
      - ./keycloak.json:/opt/keycloak/keycloak.json:ro
```
et au passage le fichier est passé en lecture seule depuis le conteneur